requirements.txt

```diff
@@ -0,0 +1,11 @@
+# older versions of these dependencies may work
+# these are simply the newest versions at the time this file was made
+joblib>=0.12.5
+numpy>=1.15.4
+# change to just tensorflow if you don't have a GPU
+tensorflow-gpu>=1.11.0
```

Comment:
The code in this repo is not compatible with the latest TF, so some restriction here is needed.

Reply:
Thanks, and good point! Unfortunately, I am no longer working with this software, so I don't have the time to figure out what an appropriate upper bound is (and this repository doesn't seem to be maintained, anyhow).